README.org

```diff
@@ -63,6 +63,9 @@
 
 ** Installation Guide
 
+*** Method 1.(Linux, MacOS)
```

Comment:
I want to finish Windows version and document it

Reply:
Ok.